### ISSUE TITLE
Editor: Not able to run the WorldEditor on Ubuntu 18.04

### ISSUE THREAD

**Issue by sawantshivaji1997** [eprikazchikov/thunder#191]
## Description

Not able to run WorldEditor
Getting the following errors
`./WorldEditor: error while loading shared libraries: libphysfs-editor.so: cannot open shared object file: No such file or directory`

## Prerequisites

--

## Steps to Reproduce

1. Build the code as described in [install.rst](https://github.com/eprikazchikov/thunder-doc/blob/master/basics/install.rst)
2. Run the WorldEditor

### Expected behavior:

WorldEditor should resolve all dependencies on its own and should successfully run
Basically, what I meant to say is that all required shared object files path should be included in LD_LIBRARY_PATH while building the code

### Actual behavior:

Gave error `./WorldEditor: error while loading shared libraries: libphysfs-editor.so: cannot open shared object file: No such file or directory`

### Reproducibility:

Tried only one time

## Versions

Hit the error in Master branch
OS: Ubuntu 18.04 LTS bionic 
QT- 5.12.3
qbs- 1.11.0

## Additional Information

--

**Comment by eprikazchikov** [comment]
Please test a patch set

**Comment by sawantshivaji1997** [comment]
everything working fine with the new patch
